.github/workflows/release-app.yml

```diff
@@ -129,6 +131,7 @@ jobs:
           key: ${{ runner.os }}-pre-commit-${{ hashFiles('.pre-commit-config.yaml') }}
 
       - name: Lint the changelog file so that it passes CI
+        # Temporarily uninstall pre-commit so the changelog PR doesn't fail
```

Comment:
Does this comment actually mean anything? I don't see any corresponding changes.

Reply:
Shoot! I'll remove